kombu/transport/qpid.py

```diff
@@ -96,41 +99,51 @@
     fcntl = None  # noqa
 
 try:
-    import qpidtoollibs
+    from qmf.client import BrokerAgent, ReconnectDelays
 except ImportError:  # pragma: no cover
-    qpidtoollibs = None     # noqa
+    BrokerAgent = None     # noqa
+    ReconnectDelays = None  # noqa
 
 try:
-    from qpid.messaging.exceptions import ConnectionError, NotFound
-    from qpid.messaging.exceptions import Empty as QpidEmpty
-    from qpid.messaging.exceptions import SessionClosed
+    import qpid
 except ImportError:  # pragma: no cover
-    ConnectionError = None
-    NotFound = None
-    QpidEmpty = None
-    SessionClosed = None
+    qpid = None
 
 try:
-    import qpid
+    import proton
+    from proton.handlers import MessagingHandler
+    from proton.reactor import Container
+    from proton.utils import ConnectionClosed, ConnectionException
 except ImportError:  # pragma: no cover
-    qpid = None
+    proton = None
+    MessagingHandler = object
+    Container = None
+    ConnectionClosed = None
+    ConnectionException = None
 
 
 from kombu.five import Empty, items, monotonic
 from kombu.log import get_logger
 from kombu.transport.virtual import Base64, Message
 from kombu.transport import base
 
-
 logger = get_logger(__name__)
 
 
 OBJECT_ALREADY_EXISTS_STRING = 'object already exists'
 
-VERSION = (1, 0, 0)
-__version__ = '.'.join(map(str, VERSION))
+PROTON_PERIOD = 0.25 # The waiting period between ProtonThread check-ins
 
-PY3 = sys.version_info[0] == 3
+
+StartConsumer = namedtuple('StartConsumer', ['queue'])
```

Comment:
I think a comment here would help to describe what these named tuples are for.

Reply:
Great idea. I just pushed a change with docstrings for each object type.